### DISCUSSION TITLE
Speed mods are not reflected in the song's length on carousel screen

### DISCUSSION THREAD

**Discussion by marioman63** [ppy/osu#25678] [Ideas] [answered]
Unlike stable, when enabling half/double time, the length of the song displayed on the left of the screen displayed on the left of the screen does not reflect the change in length caused by the mods (note that BPM is appropriately adjusted in both lazer and stable, however):

stable behaviour (no mod above, DT below):
![image](https://github.com/ppy/osu/assets/76608775/72a357f5-a634-48f0-9ee9-06d9f1773bdf)

Lazer behaviour (no speed mod left, DT right):
![image](https://github.com/ppy/osu/assets/76608775/ebb24df0-404a-4ae3-86e7-9b59c30c12b4)

**Comment by cl8n** [comment]
would be nice to indicate a change has occurred too, like how stable goes from white to red text in the DT example

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/10580
